Jetchat/app/src/main/java/com/example/compose/jetchat/DroidconEmbeddingsWrapper.kt

```diff
@@ -196,19 +199,19 @@ class DroidconEmbeddingsWrapper(val context: Context?) {
                     handled = false
                     chatResponse += " " + function.name + " " + function.arguments
                     conversation.add(
-                        ChatMessage(
+                        CustomChatMessage(
                             role = ChatRole.Assistant,
-                            content = chatResponse
+                            userContent = chatResponse
                         )
                     )
                 }
             }
             if (handled) {
                 // add the 'call a function' response to the history
                 conversation.add(
-                    ChatMessage(
+                    CustomChatMessage(
                         role = completionMessage.role,
-                        content = completionMessage.content
+                        userContent = completionMessage.content
                             ?: "", // required to not be empty in this case
```

Comment:
is this still true with the custom chat message class? or can we remove the null check now